src/main.c

```diff
@@ -85,22 +85,28 @@ void set_macos_bundle_resources(lua_State *L);
 #ifndef LITE_ARCH_TUPLE
   #if __x86_64__ || _WIN64 || __MINGW64__
     #define ARCH_PROCESSOR "x86_64"
+  #elif __i386__
+    #define ARCH_PROCESSOR "x86"
```

Comment:
@adamdharrison implemented the arch tuple scheme using clang way as a reference, looking at [clang documentation](https://clang.llvm.org/docs/CrossCompilation.html#target-triple) i386 seems to be an arch of its own so I wonder if instead of `x86` this should just be `i386` since there is also `i686`

Reply:
I don't think the difference matters here.

i386, i486 and i686 are all the same instruction set with different extensions
and since Lite-XL doesn't make use of any of those extensions directly (though a compiler may use them for optimization) its fine to generalize as x86

Reply:
Sounds reasonable enough, I guess native plugin developers if interested in supporting x86 thru continuous builds should target i386 for max x86 compatibility. Also maybe not a lot of people will use this editor on 32 bit platforms. Lets merge this then.